lib/events/dynamoevents/dynamoevents_test.go

```diff
@@ -147,27 +153,26 @@ func (s *DynamoeventsSuite) TestSizeBreak(c *check.C) {
 		}
 	}
 
-	lastTime := s.Clock.Now().UTC().Add(time.Hour)
+	lastTime := tt.suite.Clock.Now().UTC().Add(time.Hour)
 
 	for _, event := range events {
-		c.Assert(event.GetTime().Before(lastTime), check.Equals, true)
+		require.Equal(t, event.GetTime().Before(lastTime), true)
```

Comment:
A rewrite rule for `require.Equal(t, a, true) -> require.True(t, a)` would clean a lot of these up.